config/sets/symfony/symfony43.php

```diff
@@ -12,11 +12,13 @@
 use Rector\Renaming\Rector\Name\RenameClassRector;
 use Rector\Renaming\ValueObject\MethodCallRename;
 use Rector\Symfony\Rector\MethodCall\MakeDispatchFirstArgumentEventRector;
+use Rector\Symfony\Rector\MethodCall\WebTestCaseAssertIsSuccessfulRector;
 use Rector\Symfony\Rector\MethodCall\WebTestCaseAssertResponseCodeRector;
 
 # https://github.com/symfony/symfony/blob/4.4/UPGRADE-4.3.md
 return static function (RectorConfig $rectorConfig): void {
     # https://symfony.com/blog/new-in-symfony-4-3-better-test-assertions
+    $rectorConfig->rule(WebTestCaseAssertIsSuccessfulRector ::class);
```

Comment:
surplus space before `::` need to be removed:

```suggestion
    $rectorConfig->rule(WebTestCaseAssertIsSuccessfulRector::class);
```

Reply:
Done